service/provider/src/main/java/gov/va/vro/service/provider/camel/SlipClaimSubmitRouter.java

```diff
@@ -19,6 +20,17 @@ public class SlipClaimSubmitRouter {
   private static final long DEFAULT_REQUEST_TIMEOUT = 60000;
   public static final String NO_DIAGNOSTIC_CODE_ERROR = "No diagnostic code in properties.";
 
+  public static final Map<String, String> diagnoses;
+
+  static {
```

Comment:
Can you use the diagnosis lookup map `java/gov/va/vro/service/provider/services/DiagnosisLookup.java`?